api/src/db-connection.ts

```diff
@@ -257,9 +261,15 @@ export class Connection {
         ...relationships,
       },
     };
+
+    if (isSymbol(entity)) {
+      entity.attributes.disambiguated_id = entity.attributes.mathml
+    }
+
+    return entity
```

Comment:
```suggestion
    return entity;
```

Reply:
This is totally optional because I just scrolled ahead and realized that yes auto-formatting would be a very useful thing for this repo 😔